WordPress/src/main/java/org/wordpress/android/ui/people/PersonDetailFragment.java

```diff
@@ -0,0 +1,90 @@
+package org.wordpress.android.ui.people;
+
+import android.app.Fragment;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import org.wordpress.android.R;
+import org.wordpress.android.datasets.PeopleTable;
+import org.wordpress.android.models.Person;
+import org.wordpress.android.models.Role;
+import org.wordpress.android.util.GravatarUtils;
+import org.wordpress.android.widgets.WPNetworkImageView;
+
+public class PersonDetailFragment extends Fragment {
+    private static String ARG_PERSON_ID = "person_id";
+    private static String ARG_LOCAL_TABLE_BLOG_ID = "local_table_blog_id";
+
+    private long mPersonID;
+    private int mLocalTableBlogID;
+
+    private WPNetworkImageView mAvatarImageView;
+    private TextView mDisplayNameTextView;
+    private TextView mUsernameTextView;
+    private TextView mRoleTextView;
+    private TextView mRemoveTextView;
+
+    public static PersonDetailFragment newInstance(long personID, int localTableBlogID) {
+        PersonDetailFragment personDetailFragment = new PersonDetailFragment();
+        Bundle bundle = new Bundle();
+        bundle.putLong(ARG_PERSON_ID, personID);
+        bundle.putInt(ARG_LOCAL_TABLE_BLOG_ID, localTableBlogID);
+        personDetailFragment.setArguments(bundle);
+        return personDetailFragment;
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        final ViewGroup rootView = (ViewGroup) inflater.inflate(R.layout.person_detail_fragment, container, false);
+
+        mPersonID = getArguments().getLong(ARG_PERSON_ID);
+        mLocalTableBlogID = getArguments().getInt(ARG_LOCAL_TABLE_BLOG_ID);
+
+        mAvatarImageView = (WPNetworkImageView) rootView.findViewById(R.id.person_avatar);
+        mDisplayNameTextView = (TextView) rootView.findViewById(R.id.person_display_name);
+        mUsernameTextView = (TextView) rootView.findViewById(R.id.person_username);
+        mRoleTextView = (TextView) rootView.findViewById(R.id.person_role);
+        mRemoveTextView = (TextView) rootView.findViewById(R.id.person_remove);
+
+        return rootView;
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+
+        refreshPersonDetails();
+    }
+
+    public void refreshPersonDetails() {
+        if (!isAdded()) return;
+
+        Person person = PeopleTable.getPerson(mPersonID, mLocalTableBlogID);
+        if (person != null) {
```

Comment:
Interesting. Since we're guarding for null here, it might also make sense to log this in case it is null. Wdyt?